src/Symfony/Component/Messenger/Command/FailedMessagesShowCommand.php

```diff
@@ -41,7 +41,7 @@ protected function configure(): void
             ->setDefinition([
                 new InputArgument('id', InputArgument::OPTIONAL, 'Specific message id to show'),
                 new InputOption('max', null, InputOption::VALUE_REQUIRED, 'Maximum number of messages to list', 50),
-                new InputOption('transport', null, InputOption::VALUE_OPTIONAL, 'Use a specific failure transport', self::DEFAULT_TRANSPORT_OPTION),
```

Comment:
Can we remove the DEFAULT_TRANSPORT_OPTION const (same for the other command?)

Reply:
Hi! 
I've refactored it a little - moved `DEFAULT_TRANSPORT_OPTION` into abstract class and use it for autocompletion too. With previous one it is possible to introduce backward incompatibility in some edge cases. It would be nice to unify way of handling `transport` option in `FailedMessagesRemoveCommand` and two other commands for version 6.1.